oeps/oep-0027-course-metadata-authoring.rst

```diff
@@ -0,0 +1,151 @@
+    * It is bound to a specific learner.
+
+* Given the above distinctions, each `edX subdomain`_ will manage its own concept. Hence, the authoring logic
+  implementation for each concept will also remain separate. This prevents us from needing to maintain these concepts across
+  multiple domains: keep the authoring of Discoverable-Course (**Catalog Course**) within Discovery; keep the authoring of 
+  Learning-Content-Course (**Course Run**) within Content Authoring. For example, concepts of price, marketing-publish-date,
+  marketing sales, discounts, etc., do not need to bleed into the Content Authoring service.
+
+* The following is Course Run metadata and its editing should be owned by the `Content Authoring subdomain`_:
+
+  * course run dates
+
+    * start
+    * end
+    * enrollment start
+    * enrollment end
+    * certificate availability
+
+  * pacing type
+
+* The following is Catalog Course metadata and its editing should be owned by the `Discovery subdomain`_:
+
+  * course about image
+  * instructors
+  * seat price
+  * seat type
+  * other catalog course marketing data
+
+  **Note:** course about image and instructors are currently managed as Course Run metadata but will transition to be Catalog Course metadata. 
+
+* The Discovery service will be the single source of truth for the published version of the above data. Though to address the data
+  synchronization issues, we will limit the editing of a piece of data to its single owning service. Anywhere else that a piece of data is editable
+  will be made read-only.
+
+* We will introduce a new front end for editing Catalog Course metadata, written using the latest recommended front end technologies and leveraging phase 3
+of the `decoupled frontend architecture`_ plan.
+
+* From the course author's perspective, we will provide a unified user experience by having the current Studio tool
+  be the portal to authoring both Course Run metadata and Catalog Course metadata. So even though authoring of Catalog Course
+  metadata will live in the Discovery subdomain, there will be a seamless transition between the front ends of Course Run
+  Authoring and Catalog Course Authoring.
+
+  * This will depend on upcoming architectural infrastructure work on single sign on across the edX microservices and a universal
+    front end header and footer.
+
+  .. _decoupled frontend architecture: https://openedx.atlassian.net/wiki/spaces/FEDX/pages/790692200/Decoupled+Frontend+Architecture
+  .. _Discovery subdomain: https://openedx.atlassian.net/wiki/spaces/AC/pages/663224968/edX+DDD+Bounded+Contexts#edXDDDBoundedContexts-edXSubdomainMap
+  .. _Content Authoring subdomain: https://openedx.atlassian.net/wiki/spaces/AC/pages/663224968/edX+DDD+Bounded+Contexts#edXDDDBoundedContexts-edXSubdomainMap
+  .. _edX subdomain: https://openedx.atlassian.net/wiki/spaces/AC/pages/663224968/edX+DDD+Bounded+Contexts#edXDDDBoundedContexts-edXSubdomainMap
+
+.. image:: oep-0027/publisher_v3_future.png
```

Comment:
This is a good image, but part of the doc should help the reader transition from subdomain/bounded context language to our application/service names.  Maybe the contexts could be included with dashed lines?  Maybe text outside of the image?